source/deployment/cluster.rst

```diff
@@ -265,6 +264,20 @@ In Mattermost v4.2 and later, a cluster leader election process assigns any sche
 
 The process is based on a widely used `bully leader election algorithm <https://en.wikipedia.org/wiki/Bully_algorithm>`_ where the process with the lowest node ID number from amongst the non-failed processes is selected as the "leader". 
 
+Job Server
+^^^^^^^^^^^^^^^^
+
+Mattermost schedules and completes periodic tasks via the `job server <https://docs.mattermost.com/administration/config-settings.html#jobs>`_. These tasks include:
+
+ - LDAP sync
+ - Data retention
+ - Compliance exports
+ - Elasticsearch indexing
+
+When running in High Availablity mode, the ``JobSettings.RunScheduled`` setting should be set to ``false`` in config.json. This will make all the app servers also be Job Servers, allowing the High Availability cluster leader election process to run only a single scheduler. Otherwise, all the app servers run the scheduled jobs causing the jobs to be run multiple times (once per app server).
```

Comment:
The assertion that it makes them become job servers is not actually implied by this setting. I need to think about how to word this better when I've investigated the above comment.

Reply:
Given the comment I've made on the config setting, this paragraph now makes no sense so should be removed.